src/kernel/include/thd.h

```diff
@@ -75,7 +77,7 @@ struct thread {
 	struct list        event_head; /* all events for *this* end-point */
 	struct list_node   event_list; /* the list of events for another end-point */
 } CACHE_ALIGNED;
-
+#include "fpu.h"
```

Comment:
You can't have fpu.h and thd.h include each other at the same time. It will cause an error.